components/pedigree/resources/src/main/resources/pedigree/model/export.js

```diff
@@ -656,6 +660,11 @@ define([
                            " year of death, if applicable, specified or estimated." +
                            " Not doing so may lead to an overestimation of risk.");
          }
+         if (warnBrcaTestMethod) {
+             warnings.push("BOADICEA requires the testing method (search vs. direct) to be specified to" +
+                           " estimate false negative rate. The testing method has been set to 'search' to" +
+                           " avoid overestimating risk, but you should review and correct this information in BOIDICEA.");
```

Comment:
Typo in `BOIDICEA`

Reply:
Thanks for catching this, I will make another commit to fix.